core/utils/wasm-builder/src/lib.rs

```diff
@@ -156,8 +156,12 @@ fn create_out_file(file_name: &str, content: String) {
 
 /// Get a cargo command that compiles with nightly
 fn get_nightly_cargo() -> Command {
-	if Command::new("rustup").args(&["run", "nightly", "cargo"])
-		.status().map(|s| s.success()).unwrap_or(false)
+	if Command::new("rustup")
+		.args(&["run", "nightly", "cargo"])
+		.stdout(Stdio::null())
```

Comment:
Would it be worth to actually put all the logs to some temporary file instead?

Reply:
This output is 100% useless, so we don't need it at all :)